src/Microsoft.AspNet.Razor/Generator/Compiler/CodeTree/Chunks/TagHelpers/TagHelperChunk.cs

```diff
@@ -29,5 +48,10 @@ public class TagHelperChunk : ChunkBlock
         /// The HTML tag name.
         /// </summary>
         public string TagName { get; set; }
+
+        /// <summary>
+        /// Indicates whether or not the tag is self-closing.
+        /// </summary>
+        public bool SelfClosing { get; set; }
```

Comment:
this property should not have a setter.  for that matter, does this class need any setters?